### PR TITLE
Rebase action: /rebase as comment

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

File: .github/workflows/rebase.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        name: Rebase
+        if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout the latest code
+                uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
+            -   name: Automatic Rebase
+                uses: cirrus-actions/rebase@1.3.1
```
Comment (AndyScherzinger) [inline .github/workflows/rebase.yml:16]: latest would be 1.5: https://github.com/cirrus-actions/rebase/releases

File: .github/workflows/rebase.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+    rebase:
+        name: Rebase
+        if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout the latest code
+                uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
```
Comment (AndyScherzinger) [inline .github/workflows/rebase.yml:14]: > NOTE: To ensure GitHub Actions is automatically re-run after a successful rebase action use a [Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) for `actions/checkout@v2` and `cirrus-actions/rebase@1.4`. See the following [discussion](https://github.community/t/triggering-a-new-workflow-from-another-workflow/16250/37) for more details.

starting with v1.4 theer seems to be a slight change with the tokens

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky see comments which might be the reason you used 1.3.1, maybe?

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky see comments which might be the reason you used 1.3.1, maybe?

I just c&p from Desktop repo.

**Review by AndyScherzinger — APPROVED** [review]
LGTM update to newer versions whenever we see fit

**Pull request merged**